packages/opencensus-propagation-b3/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@opencensus/opencensus-propagation-b3",
+  "version": "0.0.1",
+  "description": "OpenCensus is a toolkit for collecting application performance and behavior data.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.js",
+  "repository": "census-instrumentation/opencensus-node",
+  "scripts": {
+    "build": "node_modules/.bin/tsc --declaration",
+    "test": "npm run script run-unit-tests",
```

Comment:
Q: is there a reason we are not using `gts check` to enforce the style guide at this point?

Reply:
Yes, we are using ``gts check`` for all the packages. I think it was an outdated version.